lib/Model.js

```diff
@@ -45,9 +45,9 @@ async function processCondition(req, options, schema) {
         const k = keys[i];
 
         const val = options.conditionValues[k];
-        const attr = schema.attributes[k];
+        const attr = model.$__.schema.attributes[k];
```

Comment:
have you seen a miss in this pattern else where?

Reply:
Not that I've noticed but I did see two instances of what looks like an incorrect call to `toDynamo`.

https://github.com/dynamoosejs/dynamoose/blob/9fd901a4a1755cdd19cdc808276a6895c8cc9434/lib/Model.js#L785
and
https://github.com/dynamoosejs/dynamoose/blob/9fd901a4a1755cdd19cdc808276a6895c8cc9434/lib/Model.js#L1527